File: src/JuliaExcelBridge/startup.jl
using LinearAlgebra

hello_world() = "Hello from Julia"

add_numbers(x, y) = x + y

make_seq_table(from, to; step = 1) = [collect(from:step:to) (collect(from:step:to) .^ 2)]

include("JuliaFunctions.jl")

File: src/PythonExcelBridge/startup.py
import math


def hello_world():
    return "Hello from Python"


def add_numbers(x, y):
    return x + y


def make_seq_table(start, stop, step=1):
    xs = list(range(int(start), int(stop) + 1, int(step)))
    return [[x, x * x] for x in xs]


try:
    import pandas as pd  # noqa: F401
except Exception:
    pass

from PythonFunctions import *
